app/views/conferences_primer_theme/pages/about.html.haml

```diff
@@ -13,6 +13,9 @@
     - cache [partable_for(:minutes), content(:minutes)&.structure_items,
              content(:minutes)&.structure_items&.collect_concat(&:structure_parts)&.collect(&:structure_partable)] do
       .mt-4.col-12.col-md-6.float-md-left= render partial: 'minutes', object: content(:minutes)
+    - cache [partable_for(:useful_documents), content(:useful_documents)&.structure_items,
+             content(:useful_documents)&.structure_items&.collect_concat(&:structure_parts)&.collect(&:structure_partable)] do
+      .mt-4.col-12.col-md-6.float-md-left= render partial: 'useful_documents', object: content(:useful_documents)
```

Comment:
With the partial you're rendering here (and the partial for the individual document), it's good practice to have the inner `.html` extension in case we ever need to add a non-html partial (`.haml` alone doesn't determine the format).

Reply:
Whoops yes typo on my part.